src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -60,14 +60,12 @@
       @case "RequestPagedAll"
         {@pagedOptionalArrayMethodSampleCodeAll(apiMethod, sample)}
       @case "RequestStreamingBidi"
-        {@optionalArrayMethodSampleCodeBidiStreaming(apiMethod, sample.initCode)}
-        {@processResponse(sample)}
+        {@optionalArrayMethodSampleCodeBidiStreaming(apiMethod, sample)}
```

Comment:
For clarity/consistency, I would suggest moving the remaining `processResponse` calls out of the `switch` and into the functions being called.